fe2-android/app/src/main/java/com/github/dedis/popstellar/utility/handler/data/RollCallHandler.java

```diff
@@ -138,23 +132,16 @@ public void handleCloseRollCall(HandlerContext context, CloseRollCall closeRollC
     lao.updateWitnessMessage(messageId, closeRollCallWitnessMessage(messageId, rollCall));
 
     // Subscribe to the social media channels
-    try {
-      PoPToken token = keyManager.getValidPoPToken(laoView.getId(), rollCall);
-      context
-          .getMessageSender()
-          .subscribe(channel.subChannel("social").subChannel(token.getPublicKey().getEncoded()))
-          .subscribe(
-              () -> Log.d(TAG, "subscription a success"),
-              error -> Log.d(TAG, "subscription error"));
-
-    } catch (InvalidPoPTokenException e) {
-      Log.i(TAG, "Received a close roll-call that you did not attend");
-    } catch (KeyException e) {
-      Log.e(
-          TAG,
-          "Could not retrieve your PoP Token to subscribe you to your social media channel",
-          e);
-    }
+    rollCall
+        .getAttendees()
+        .forEach(
+            token ->
+                context
+                    .getMessageSender()
+                    .subscribe(channel.subChannel("social").subChannel(token.getEncoded()))
+                    .subscribe(
+                        () -> Log.d(TAG, "subscription a success"),
+                        error -> Log.d(TAG, "subscription error")));
```

Comment:
The logic here changes rather brutally.

It's my understanding in the past (at least on Fe1) that you'd have to choose who you subscribe to.
So by default, you'd only subscribe to the main social channel, but then you could choose to subscribe to some, all or no PoP token.

This logic auto-subscribes to all the PoP tokens. Is it correct / what you intended?